Automattic/wp-calypso
# Add "Try again" button for product form categories

This adds a "Try again" button for if the product categories finally fail to load on the product create page.

To Test:

1. `npm start`
2. Visit the product create page.
3. Disable your WooCommerce Plugin.
4. Refresh the product create page. After 3 requests, the data layer will stop trying to fetch the categories and a warning should pop up with a "Try again" button.
